update.md

```diff
@@ -0,0 +1,54 @@
+# ESI demo environment setup
+## Requirements
+* CentOS 8
+* Ussuri tripleo-repos
+## Steps
+* Download and install the python-tripleo-repos RPM
+```
+sudo dnf install -y https://trunk.rdoproject.org/centos8/component/tripleo/current/python3-tripleo-repos-0.1.1-0.20200612051255.ecf6206.el8.noarch.rpm
+```
+* Enable the current Ussuri repositories
+```
+sudo -E tripleo-repos -b ussuri current
+```
+* Install the TripleO CLI
+```
+sudo yum install -y python3-tripleoclient
+```
+* git clone esi-sandbox repo and cd esi-sandbox
+* Open deploy.sh, change INTERFACE=eth0 to INTERFACE=ens10; comment out ceph-ansible related commands.
+* Open standalone_parameters.yaml
+    * remove resource_registry: DockerRegistry, MistralEngine, MistralApi, MistralExecutor, MistralEventEngine
+    * change ContainerImagePrepare namespace to docker.io/tripleou
+    * change NeutronPublicInterface value to ens10   
+* Run ./deploy.sh
+* After deploy successes, 'export OS_CLOUD=standalone' before running OpenStack commands.
+* Cleanup a deployment: run the following script
+```
+#!/bin/bash
+echo "Tearing down TripleO environment"
+if type pcs &> /dev/null; then
+    sudo pcs cluster destroy
+fi
+if type podman &> /dev/null; then
+    echo "Removing podman containers and images (takes times...)"
+    sudo podman rm -af
+    sudo podman rmi -af
+fi
+sudo rm -rf \
+    /var/lib/tripleo-config \
+    /var/lib/config-data /var/lib/container-config-scripts \
+    /var/lib/container-puppet \
+    /var/lib/heat-config \
+    /var/lib/image-serve \
+    /var/lib/containers \
+    /etc/systemd/system/tripleo* \
+    /var/lib/mysql/*
+sudo systemctl daemon-reload
```

Comment:
This is an *excellent* thing to include! Can you actually make it a script in the repository, and just mention that script here?

Reply:
Ok!